### PR TITLE
Title for Pull Request

### PR DESCRIPTION
**Pull request by lowfive-servo**
Test

### CODE REVIEW

File: traviscibuildfail.py [hunk 1]
```diff
@@ -58,4 +61,4 @@ def handle_payload(payload, username, token):
```
Comment (lowfive-servo) [inline traviscibuildfail.py]: What is this?

File: traviscibuildfail.py [hunk 2]
```diff
@@ -58,4 +61,4 @@ def handle_payload(payload, username, token):
     payload = json.loads(open('test/test-data.json').read())
```
Comment (lowfive-servo) [inline traviscibuildfail.py]: What is this?

Comment (lowfive-servo) [inline traviscibuildfail.py]: What is this?

Comment (JoshTheGoldfish) [inline traviscibuildfail.py]: This looks odd

Comment (lowfive-servo) [inline traviscibuildfail.py]: Just Testing?

Comment (lowfive-servo) [inline traviscibuildfail.py]: Just Testing?

**Pull request closed without merging**